.ci/docker.run

```diff
@@ -8,7 +8,7 @@
 set -e
 
 source $TRAVIS_BUILD_DIR/.ci/docker-prelude.sh
-
+apt-get install -y curl
```

Comment:
This should go over to our container image: https://github.com/tpm2-software/tpm2-software-container

Reply:
Done.